pkcs11/defaults.py

```diff
@@ -44,6 +45,7 @@
     KeyType.EC: _SIGNING | MechanismFlag.DERIVE,
     KeyType.RSA: _ENCRYPTION | _SIGNING | _WRAPPING,
     KeyType.GENERIC_SECRET: 0,
+    KeyType.EC_EDWARDS: _SIGNING,
```

Comment:
DERIVE ?

Reply:
For deriving, you need EC_MONTGOMERY (aka X25519). Ed25519/EC_EDWARDS only does signatures. I think SoftHSM doesn't have X25519, so I can't test.